build/shade/_use-volatile-feed.shade

```diff
@@ -10,9 +11,12 @@ use namespace="System.Xml.Linq"
         .Select(entry => new KeyValuePair<string, string>(((string)entry.Key).Substring(prefix.Length), (string)entry.Value))
         .ToList();
 
-    if (feeds.Any())
+    var nugetConfigPath = Directory.EnumerateFiles(Directory.GetCurrentDirectory())
```

Comment:
Do you know what repos have files that are differently cased? Perhaps we could fix them instead. dotnet also expects specifying casing.